apps/preview/next/pages/showcases/Search/SearchBasic.tsx

```diff
@@ -138,14 +143,17 @@ export default function SearchBasic() {
         }
       />
       {isOpen && (
-        <div ref={refs.setFloating} style={style} className="right-0 left-0">
+        <div ref={refs.setFloating} style={style} className="left-0 right-0">
```

Comment:
suggestion: please do remove headwing, this plugin for tailwind ordering class, there was once discussion about it, I removed it and for now IMHO we should not have something like that, I mean if we will have extensions like that we should have it all or not at all. But then we need to document it etc

Reply:
Ok, I removed it, but I'll leave this change like it is, if it's okay?

Reply:
Sure, we can keep it, thank you :)